pkg/pit/dnsmasq.go

```diff
@@ -149,6 +149,32 @@ host-record={{.APIGWALIASES}},{{.APIGWIP}} # api gateway
 cname=kubernetes-api.vshasta.io,ncn-m001
 `)
 
+// StaticConfigTemplateCHN defines the DNSMasq configuration for a node if a system is configured to use CHN and there is no CAN network.
+var StaticConfigTemplateCHN = []byte(`
+# Static Configurations
+{{range .NCNS}}
+# DHCP Entries for {{.Hostname}}
+dhcp-host=id:{{.Xname}},set:{{.Hostname}},{{.Bond0Mac0}},{{.Bond0Mac1}},{{.MtlIP}},{{.Hostname}},20m # MTL
+dhcp-host=id:{{.Xname}},set:{{.Hostname}},{{.Bond0Mac0}},{{.Bond0Mac1}},{{.NmnIP}},{{.Hostname}},20m # Bond0 Mac0/Mac1
+dhcp-host=id:{{.Xname}},set:{{.Hostname}},{{.Bond0Mac0}},{{.Bond0Mac1}},{{.HmnIP}},{{.Hostname}},20m # HMN
+dhcp-host={{.BmcMac}},{{.BmcIP}},{{.Hostname}}-mgmt,20m #HMN
+# Host Record Entries for {{.Hostname}}
+host-record={{.Hostname}},{{.Hostname}}.hmn,{{.HmnIP}}
```

Comment:
Do we need `.chn` entries (mirroring the other groups here)?

Reply:
I'm not sure that we do, while we know what the IPs are ahead of time they aren't present on any interface until Slingshot is installed and configured and the appropriate Ansible has been run by CFS.

All of this happens long after we've shutdown dnsmasq on the PIT.